### ISSUE TITLE
Can we bring our laptop with preferred keyboard to the exam?

### ISSUE THREAD

**Issue by egejjespersen** [snu-sf/pl2015#163]
Hello, @jeehoonkang !

Can we bring our laptop with preferred keyboard to the exam, just as we did to the mid-term?

Kind regards,
Ege

**Comment by jeehoonkang** [comment]
Yes. But due to space limitation I may have to let only 4-5 people to take
exams in laptop.

2015-06-17 2:46 GMT-07:00 egejjespersen notifications@github.com:

> Hello, @jeehoonkang https://github.com/jeehoonkang !
> 
> Can we bring our laptop with preferred keyboard to the exam, just as we
> did to the mid-term?
> 
> Kind regards,
> Ege
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/snu-sf/pl2015/issues/163.

## 

Jeehoon Kang (Ph.D. student) http://sf.snu.ac.kr/jeehoon.kang
Software Foundations Laboratory http://sf.snu.ac.kr
Seoul National University http://www.snu.ac.kr

**Comment by Parkdaeyoung** [comment]
I use Proof-general and I brought my lap-top on mid exam. May I bring one this time?

**Comment by jaewooklee93** [comment]
@jeehoonkang
When do we take the exam?

**Comment by Parkdaeyoung** [comment]
@jaewooklee93 I guess that the exam will be on 6:00-9:00 p.m, today.

**Comment by jeehoonkang** [comment]
today 1800-2100.
Take your laptops. But I cannot gurantee you can use..

**Comment by jaewooklee93** [comment]
@Parkdaeyoung @jeehoonkang  Thank you.
